pkgs/development/libraries/vtk/generic.nix

```diff
@@ -23,6 +24,18 @@ in stdenv.mkDerivation {
     sha256 = sourceSha256;
   };
 
+  preConfigure = let
```

Comment:
maybe consider `postPatch` here?

Reply:
Can do that, but why is it preferable? :)

Reply:
Just seems more related to unpacking or patching of sources than configuring.

Reply:
Moved to postPatch.